### PR TITLE
Update HK Content Modules

### PR DESCRIPTION
**Pull request by juanpedromoreno**
I'm suggesting some `Lorem ip amet` replacements.

### CODE REVIEW

File: _data/features.yml [hunk 1]
```diff
@@ -1,12 +1,12 @@
 content:
     - title: Purely functional
-      description: Lorem ip amet, consectetur adipiscing elit. Nullam blandit odio nec button amet, consectetur adip.
+      description: Functional programs and libraries targeted to improve parallel computations, eager evaluation, and the use purely data structures.
```
Comment (calvellido) [inline _data/features.yml]:
```suggestion
      description: Functional programs and libraries targeted to improve parallel computations, eager evaluation, and the use of pure data structures.
```

File: _data/features.yml [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
       icon: /img/higherkindness-feature-icon-primary.svg
 
     - title: Modulable
-      description: Lorem ip amet, consectetur adipiscing elit. Nullam blandit odio nec button amet, consectetur adip.
+      description: Combining simple libraries to build more complicated ones.
```
Comment (calvellido) [inline _data/features.yml]:
```suggestion
      description: Combining simple libraries to build more complex ones.
```

**Comment by juanpedromoreno** [comment]
Thanks @calvellido

**Comment by raulraja** [comment]
@andyscott ^^^ any thoughts on these blurbs for the HK landing?

File: _data/features.yml [hunk 3]
```diff
@@ -1,12 +1,12 @@
...
     - title: Modulable
-      description: Lorem ip amet, consectetur adipiscing elit. Nullam blandit odio nec button amet, consectetur adip.
+      description: Combining simple libraries to build more complex ones.
       icon: /img/higherkindness-feature-icon-secondary.svg
 
     - title: Safety
-      description: Lorem ip amet, consectetur adipiscing elit. Nullam blandit odio nec button amet, consectetur adip.
-      icon: /img/higherkindness-feature-icon-tertiary.svg
\ No newline at end of file
+      description: Preserving the referential transparency and type safety.
```
Comment (andyscott) [inline _data/features.yml]: Nit: remove "the"

**Comment by andyscott** [comment]
One nit. Otherwise, LGTM.

**Pull request merged**